apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/datasources.md

```diff
@@ -47,194 +32,7 @@ In order to complete the following tutorial, you will need these resources:
-   </Alert>
-
-1. Select the Folder icon from the options shown.
-
-   <Image title="Content Editor - Select Folder icon" src="https://sitecorecontenthub.stylelabs.cloud/api/public/content/1b93d1b6a6814d27a61bd142aefcc792?v=b3346dfb" maxW="xl" />
-
-### Configure Insert Options
-
-When an author tries to create a new content item, they are presented with the types of content that are allowed at that location. [Insert Options](https://doc.sitecore.com/xmc/en/developers/xm-cloud/insert-options.html) allow you to specify what content options are available to the author. You will now configure the options for a Text Teaser Folder so that Authors can easily create new Text Teaser content items or create more Text Teaser Folders to structure their content even further.
-
-   <Alert status="info">
-   <AlertIcon />
-   In this section you will get a first look at Sitecore XM Cloud "Standard Values" for templates. The Standard Values on a template usually contain default values that are automatically set on content creation but can be updated by the author later. If you're interested in learning more about Standard Values, check out the documentation: https://doc.sitecore.com/xmc/en/developers/xm-cloud/standard-values-for-data-template-fields.html
-   </Alert>
-
-1. In the content tree, select the **Text Teaser Folder** template.
-1. In the ribbon, switch to the **Builder Options** tab
-
-   <Image title="Content Editor - Open Builder Options" src="https://sitecorecontenthub.stylelabs.cloud/api/public/content/05be5f1c4c9e4480979b841a60f0b36a?v=6f3df81a" maxW="xl" />
-
-1. Click on the **Standard values** button to create a new definition for the template. A new item named `__Standard Values` will be created and the UI will automatically update to select this new item.
-1. In the ribbon, switch to the **Configure** tab. You can now configure options for the \_\_Standard Values item.
-1. Click the **Assign** button in the "Insert Options" ribbon chunk to launch the Insert Options dialog. This dialog allows us to select which templates can be inserted when the user has a Text Teaser Folder selected.
-
-   <Image title="Content Editor - Assign insert options" src="https://sitecorecontenthub.stylelabs.cloud/api/public/content/af5561ec5826446cb52f75b7c381e109?v=891bce56" maxW="xl" />
-
-1. In the selection tree on the left, navigate to `/Templates/Project/Company Dev/Basic Components`
-1. Double-click on the **Text Teaser** template to add it to the **Selected** pane on the right
-1. Double-click on the **Text Teaser Folder** template to add it to the **Selected** pane on the right
-
-   <Image title="Content Editor - Select Insert Options" src="https://sitecorecontenthub.stylelabs.cloud/api/public/content/83ca1438aef7402491b87738e61dfb81?v=56fb9f4b" maxW="xl" />
-
-1. Click the **OK** button to save the changes.
```

Comment:
I wonder if we should add a screenshot of the component using the data from XM Cloud

Reply:
I'm not sure how it got tagged to this part of the doc, but I agree with your comment that at the very end of the tutorial, when the datasource item is selected and we are supposed to see the Home Page update we are missing a screenshot. Do you think you could make that screenshot happen and add it right before the Congratulations message?